queue_services/entity-emailer/tests/unit/email_processors/test_nr_notification.py

```diff
@@ -33,7 +33,7 @@
         [{'name': 'TEST Company Name', 'state': 'NE'}, {'name': 'TEST2 Company Name', 'state': 'APPROVED'}]),
     ('before-expiry', 'NR 1234567', 'Expiring Soon', '2021-07-20T00:00:00+00:00', None, 'TEST3 Company Name',
         [{'name': 'TEST3 Company Name', 'state': 'CONDITION'}, {'name': 'TEST4 Company Name', 'state': 'NE'}]),
-    ('expired', 'NR 1234567', 'Expired', None, None, None,
+    ('expired', 'NR 1234567', 'Expired', None, None, 'TEST Expired Company Name',
```

Comment:
I think code needs to be added to the test to include a block where option is `expired` so you can actually check the expected legal name

Reply:
I had to create a new P.R. i saw this after the merge.

Please review new P.R.  1402

https://github.com/bcgov/lear/pull/1402